datalab/requirements-datalab.txt

```diff
@@ -0,0 +1,2 @@
+git+https://github.com/elisno/cleanlab.git@datalab#egg=cleanlab[datalab]
```

Comment:
don't forget to update this when next version of cleanlab is released that contains Datalab